src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -2,21 +2,23 @@
  * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
  */
 
+import {isUndefined, get} from 'lodash';
 import {take} from 'redux-saga/effects';
-import {get} from 'lodash';
 
 import {LO_VIEW, LO_MODE, SHOW_DROPDOWN, SET_LAYOUT_MODE, getLayouInfo, dispatchUpdateLayoutInfo, dropDownHandler} from '../../core/LayoutCntlr.js';
 import {TBL_RESULTS_ADDED, TABLE_LOADED, TBL_RESULTS_ACTIVE, TABLE_HIGHLIGHT} from '../../tables/TablesCntlr.js';
 import {getCellValue, getTblById, makeTblRequest} from '../../tables/TableUtil.js';
 import {updateSet} from '../../util/WebUtil.js';
 import {dispatchPlotImage} from '../../visualize/ImagePlotCntlr.js';
-import WebPlotRequest from '../../visualize/WebPlotRequest.js';
+import {WebPlotRequest,TitleOptions} from '../../visualize/WebPlotRequest.js';
+import {makeWiseLcPlotRequest} from '../../metaConvert/WiseLcViewRequestList';
```

Comment:
not used, gives a problem of not found and breaks the lc.html page. Please remove unused import otherwise the app doesn't load.